libs/common/src/auth/services/token.service.ts

```diff
@@ -439,8 +450,12 @@ export class TokenService implements TokenServiceAbstraction {
     }
 
     // Platform doesn't support secure storage, so use state provider implementation
-    await this.singleUserStateProvider.get(userId, ACCESS_TOKEN_DISK).update((_) => null);
-    await this.singleUserStateProvider.get(userId, ACCESS_TOKEN_MEMORY).update((_) => null);
+    await this.singleUserStateProvider.get(userId, ACCESS_TOKEN_DISK).update(() => null, {
+      shouldUpdate: (previousValue) => previousValue !== null,
+    });
+    await this.singleUserStateProvider.get(userId, ACCESS_TOKEN_MEMORY).update(() => null, {
```

Comment:
```suggestion
    await this.singleUserStateProvider.get(userId, ACCESS_TOKEN_MEMORY).update((_) => null);
```
Not my ownership, but the linked epic describing the change clearly states that this applies to disk I/O only. Subsequently, there is no reason to apply it to memory state.